packages/gatsby/src/bootstrap/remove-stale-jobs.js

```diff
@@ -5,15 +5,15 @@ module.exports = state => {
   const actions = []
 
   // If any of our finished jobs are stale we remove them to keep our cache small
-  state.jobsV2.done.forEach((job, contentDigest) => {
+  state.jobsV2.complete.forEach((job, contentDigest) => {
     if (isJobStale(job)) {
       actions.push(internalActions.removeStaleJob(contentDigest))
     }
   })
 
   // If any of our pending jobs do not have an existing inputPath or the inputPath changed
   // we remove it from the queue as they would fail anyway
-  state.jobsV2.stale.forEach(({ job, plugin }) => {
+  state.jobsV2.incomplete.forEach(({ job, plugin }) => {
     if (isJobStale(job)) {
```

Comment:
since you renamed `stale` to `incomplete`:

- would it be better also rename the `isJobStale` to `isJobIncomplete` to avoid different namings?

Reply:
those namings are actually correct :) Because we want to check if the job is still relevant when files get removed or changed when gatsby is not running the jobs that we had aren't relevant anymore.

Reply:
ok